src/components/marketplace/order_book.tsx

```diff
@@ -27,16 +28,30 @@ interface State {
     tab: Tab;
 }
 
-const orderToRow = (order: OrderBookItem, index: number, count: number, selectedToken: Token) => {
+const orderToRow = (
+    order: OrderBookItem,
+    index: number,
+    count: number,
+    selectedToken: Token,
+    mySizeOrders: OrderBookItem[] = [],
+) => {
     const size = tokenAmountInUnits(order.size, selectedToken.decimals);
     const price = order.price.toString();
     const priceColor = order.side === OrderSide.Buy ? themeColors.green : themeColors.orange;
     const time: string = '';
     const timeColor = time ? '#000' : themeColors.lightGray;
 
+    const mySize = mySizeOrders.reduce((sumSize, mySizeItem) => {
```

Comment:
I don't think this will work if the user has more than one order with the same price, since it will only use the size of the last of those orders.

Reply:
If this is indeed a bug, please add a proper failing test before fixing it (@mariano-aguero can probably help with that if needed).